packages/golden_toolkit/test/configuration_test.dart

```diff
@@ -37,6 +37,83 @@ void main() {
       await multiScreenGolden(tester, 'this_is_expected_to_skip', skip: true);
     });
 
+    testGoldens('screenMatchesGolden method defers primeAssets to global configuration', (tester) async {
+      var globalPrimeCalledCount = 0;
+
+      GoldenToolkit.configure(
+        GoldenToolkitConfiguration(primeAssets: (WidgetTester tester, String name, Finder finder) async {
+          globalPrimeCalledCount += 1;
+          await defaultPrimeAssets(tester, name, finder);
+        }),
+      );
+
+      await tester.pumpWidgetBuilder(Image.asset('images/sample_cloudy.png'));
+
+      await screenMatchesGolden(tester, 'screen_matches_golden_defers_primeAssets');
+
+      expect(globalPrimeCalledCount, 1);
+    });
+
+    testGoldens('screenMatchesGolden method level primeAssets trumps global configuration', (tester) async {
```

Comment:
do you think we still need the method level override? 

I can't decide. It would simplify the APIs if we stuck to the global override. With the way flutter test's test_config construct works, it can be set for particular sub-folders, or someone could alter it for just a file (or for a particular test by configuring before and reverting after).

what do you think?

Reply:
Done

Reply:
Ok but then proably sth like `configuration.copyWith` would be helpful

Reply:
100% agree.